src/native/libs/System.Security.Cryptography.Native/osslcompat_111.h

```diff
@@ -79,6 +79,8 @@ int32_t X509_get_version(const X509* x509);
 int X509_set1_notAfter(X509* x509, const ASN1_TIME*);
 int X509_set1_notBefore(X509* x509, const ASN1_TIME*);
 int32_t X509_up_ref(X509* x509);
+int SSL_verify_client_post_handshake(SSL *s);
+void SSL_set_post_handshake_auth(SSL *s, int val);
```

Comment:
If no special `#if` is required for these prototypes, they should be sorted alphabetically in this block.

Reply:
Should be fixed now